core/pom.xml

```diff
@@ -502,6 +518,7 @@
         <project.build.sourceEncoding>${platform-encoding}</project.build.sourceEncoding>
         <project.reporting.sourceEncoding>${platform-encoding}</project.reporting.sourceEncoding>
         <project.reporting.outputEncoding>${platform-encoding}</project.reporting.outputEncoding>
+        <hive.version>1.2.2-bd103</hive.version>
```

Comment:
Is this available on public mvn? and why do we need to define custom hive version to do basic functionalities? it will only run in a specific environment.